hack/lib/serverless.bash

```diff
@@ -155,6 +155,10 @@ function deploy_knativeserving_cr {
     deploy_with_kourier
   fi
 
+  if [[ $ENABLE_TRACING == "true" ]]; then
+    enable_serving_tracing
```

Comment:
@pierDipi do you think Eventing should also integrate here? Afaik Eventing has its own setup for tracing tests?